simulator/src/main/java/com/github/benmanes/caffeine/cache/simulator/admission/Admittor.java

```diff
@@ -38,10 +40,41 @@
    */
   boolean admit(long candidateKey, long victimKey);
 
+  /** Records the access to the entry. (overloading to use events)*/
+  default void record(AccessEvent event) { record(event.key()); }
+
+  /**
+   * Returns if the candidate should be added to the cache and the page replacement policy's chosen
+   * victim should be removed.
+   *
+   * @param candidate the event of the newly added entry
+   * @param victim the event of the entry the policy recommends removing
+   * @return if the candidate should be added and the victim removed due to eviction
+   */
+  default boolean admit(AccessEvent candidate, AccessEvent victim) { return admit(candidate.key(),victim.key()); }
```

Comment:
Please make these cases multiple lines (we casually follow [Google's style guide](https://google.github.io/styleguide/javaguide.html)). For languages like Kotlin / Scala this is common for one-liners, but isn't in Java.

Reply:
Actually, you can ignore this. I'll go back and fix up minor style nits (line Google's use of 2-space indent) after this is merged.